ObjcVoiceQuickstart/ViewController.m

```diff
@@ -686,4 +709,46 @@ - (void)performAnswerVoiceCallWithUUID:(NSUUID *)uuid
     }
 }
 
+#pragma mark - Ringtone
+
+- (void)playRingtone {
+    NSString *ringtonePath = [[NSBundle mainBundle] pathForResource:@"ringtone" ofType:@"wav"];
+    if ([ringtonePath length] <= 0) {
+        NSLog(@"Can't find sound file");
+        return;
+    }
+    
+    self.ringtonePlayer = [[AVAudioPlayer alloc] initWithContentsOfURL:[NSURL URLWithString:ringtonePath] error:nil];
```

Comment:
pass an error and log error ?   
Also -
if (self.ringTonePlayer) {
 ...
} else {
//  log error
}